addons/pfconnector/configure.sh

```diff
@@ -41,3 +41,4 @@ if ! prompt "Should the pfconnector server TLS certificate be validated?"; then
 fi
 
 echo "FETCH_REMOTES_VIA_API=true" >> /etc/pfconnector-client.env
+echo "PFCONNECTOR_REMOTE=true" >> /etc/pfconnector-client.env
```

Comment:
We will need to rebase this PR on `devel` once #7232 is merged because this path will not be valid.